core/modules/czar/Czar.cc

```diff
@@ -52,6 +57,13 @@ namespace lsst {
 namespace qserv {
 namespace czar {
 
+Czar::Ptr Czar::_czarPtr;
+
+Czar::Ptr Czar::createCzar(std::string const& configPath, std::string const& czarName) {
+    _czarPtr.reset(new Czar(configPath, czarName));
+    return _czarPtr;
+}
+
```

Comment:
I hate singletons, could we avoid changing czar class into singleton? Also you changes small number of classes to depend on czar which is additional dependency I do not like either :) Having `LargeResultMgr` instance a member of czar looks like a right scope, can you just pass it to the objects that need it via parameters?